src/main/resources/staff/templates/document/settlementAgreement.html

```diff
@@ -0,0 +1 @@
+Place holder template
```

Comment:
I assume this is not ready yet?

Reply:
I have it done locally, but untested. Trying to get the frontend working